### PR TITLE
Cis2 add list

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Makes it easier as a user to find tokens for a given contract.

Currently, fallback metadata is added for non-mainnet networks (to be able to test). 

## Changes

- Manage token now presents a list of tokens to search from and a search field to search for an ID.
- Move "manage tokens" flow to be under the account view.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/browser-wallet/src/popup/pages/Account/AddTokens/state.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import { ContractDetails } from '@shared/utils/token-helpers';
+import { ContractTokenDetails, fetchTokensConfigure } from './utils';
+
+type TokensAtomAction = 'reset' | 'next';
+
+type TokenWithPageID = MakeOptional<ContractTokenDetails, 'metadata'> & {
+    pageId: number;
+};
+
+function resetOnUnmountAtom<V>(initial: V): PrimitiveAtom<V> {
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/AddTokens/state.ts:16]: This should probably be in a helper.

**Review by shjortConcordium — CHANGES_REQUESTED** [review]
Except for the wallet crashing if you manage without any tokens added (which I fix in my PR) and the checkbox being squashed on the setup page, it looks good.

File: packages/browser-wallet/src/popup/shared/Form/Form.scss [hunk 2]
```diff
@@ -51,7 +51,6 @@
     &__checkbox {
         display: flex;
         flex-direction: row;
-        padding: 0 rem(30px);
```
[the remark this answers is not in the document]
Comment (soerenbf) [reply packages/browser-wallet/src/popup/shared/Form/Form.scss:54]: I'm doing that already: `Setup.scss@onboarding-setup__intro-wrapper__terms-and-conditions`

File: packages/browser-wallet/src/popup/shared/i18n/da.ts [hunk 3]
```diff
@@ -43,6 +43,25 @@ const t: typeof en = {
         cost: 'Estimerede transaktionsomkostninger',
         unknown: 'Ukendt',
     },
+    tokenDetails: {
```
[the remark this answers is not in the document]
Comment (soerenbf) [reply packages/browser-wallet/src/popup/shared/i18n/da.ts:46]: hmm... weird, must have copied the wrong thing in. I'll translate

File: packages/browser-wallet/src/popup/pages/Account/AddTokens/state.ts (hunk 1 above)
Comment (soerenbf) [reply packages/browser-wallet/src/popup/pages/Account/AddTokens/state.ts]: Moved it to `store/utils`

File: packages/browser-wallet/src/popup/shared/Form/Form.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (shjortConcordium) [reply packages/browser-wallet/src/popup/shared/Form/Form.scss:54]: Then it doesn't work?

**Pull request merged**